proto/penumbra/penumbra/core/component/stake/v1/stake.proto

```diff
@@ -345,3 +347,55 @@ message EventTombstoneValidator {
   // The voting power for the validator.
   uint64 voting_power = 6;
 }
+
+message EventValidatorStateChange {
+  // The validator's identity key.
+  keys.v1.IdentityKey identity_key = 1;
+  // The new state of the validator.
+  ValidatorState state = 2;
+}
+
+message EventValidatorVotingPowerChange {
+  // The validator's identity key.
+  keys.v1.IdentityKey identity_key = 1;
+  // The new voting power of the validator.
+  num.v1.Amount voting_power = 2;
+}
+
+message EventValidatorBondingStateChange {
+  // The validator's identity key.
+  keys.v1.IdentityKey identity_key = 1;
+  // The new bonding state of the validator.
+  BondingState bonding_state = 2;
+}
+
+message EventRateDataChange {
```

Comment:
:bike: :house: :paintbrush: with apologies, a bicycle shed

```suggestion
message EventValidatorRateDataChange {
```

should this also have the `EventValidator`, since it also holds an identity key and relates to the broader lifecycle of a validator? :woman_shrugging: i don't feel strongly about it, but it feels grouped with the rest of these.